modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/UpgradeFeatureCard.kt

```diff
@@ -49,6 +49,14 @@ sealed class UpgradeFeatureCard(
                 SubscriptionTier.fromFeatureTier(Feature.UP_NEXT_SHUFFLE) == SubscriptionTier.PLUS
             -> LR.string.up_next_shuffle_plus_prompt
 
+            source == OnboardingUpgradeSource.FOLDERS -> LR.string.folders_plus_prompt
```

Comment:
This probably should use `FOLDERS_PODCAST_SCREEN` source as well. Otherwise upselling from a podcast screen doesn't display the custom message.